src/styles/blocks/page-content.scss

```diff
@@ -0,0 +1,103 @@
+.page {
+  &__content {
+    width: 1024px;
+    padding: 50px 40px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0 auto;
+
+    &-product {
+      display: flex;
+      flex-direction: column;
+      flex-basis: 200px;
```

Comment:
default value is column anyway, i think there is no need for both of these

Reply:
Noted. fixed